test/integration/common.rb

```diff
@@ -94,6 +94,7 @@ def with_lines_as_tempfile(lines = [], &block)
 
   # @yield [pid, port]
   def start_sshd_7_or_later(port = '2200', config: nil)
+    pid = nil
```

Comment:
Without pre-declaring `pid`, when `config` exists, the `pid` within `with_lines_as_tempfile` block will be out of scope, which results that statements in `ensure` part doesn't execute as expected (in other words, sshd process won't be killed).

Reply:
good catch thanks!